### PR TITLE
test: adding more acceptance test coverage for b2b errors

### PR DESCRIPTION
**Pull request by andpou**
## Description
adding test to ensure that empty and invalid transaction ID is handled correctly
adding test to ensure wrong business sector produces correct error

### CODE REVIEW

File: source/AcceptanceTests/Tests/B2BErrors/WhenTransactionIdAndMessageIdIsNotCorrectTests.cs [hunk 1]
```diff
@@ -56,16 +71,38 @@ public async Task Transaction_id_is_not_unique()
...
         var response = await AggregationRequest.AggregatedMeasureDataWithXmlPayload(payload, Token);
 
         Output.WriteLine(response);
 
-        await ErrorAsserter.AssertCorrectErrorIsReturnedAsync("00201", "The id of the message cannot be empty", response);
+        await ErrorAsserter.AssertCorrectErrorIsReturnedAsync("00202", "Transaction id cannot be empty", response);
+    }
+
+    [Fact]
+    public async Task Transaction_is_is_invalid_produces_error()
```
Comment (ebbeknudsen) [inline source/AcceptanceTests/Tests/B2BErrors/WhenTransactionIdAndMessageIdIsNotCorrectTests.cs:86]:
```suggestion
    public async Task Transaction_is_invalid_produces_error()
```

**Review by ebbeknudsen — CHANGES_REQUESTED** [review]

**Review by ebbeknudsen — APPROVED** [review]
